home.html

```diff
@@ -0,0 +1,207 @@
+          <h5>Program</h5>
+        </div>
+        <hr />
+        <ul class="mobile-ul">
+          <li class="mobile-program">
+            <img
+              class="icon-program"
+              src="./img/presentation.svg"
+              alt="Icon presentation"
+            />
+            <h6>Lecture</h6>
+            <p class="paragraph-program">
+              Liste to the models from various countries about the messages of
+              sharing and opening
+            </p>
+          </li>
+          <li class="mobile-program">
+            <img
+              class="icon-program"
+              src="./img/exhibition.svg"
+              alt="Icon presentation"
+            />
+            <h6>Exhibition</h6>
+            <p class="paragraph-program">
+              Listen to the models from various countries about the messages of
+              sharing and opening
+            </p>
+          </li>
+          <li class="mobile-program">
+            <img
+              class="icon-program"
+              src="./img/discussion.svg"
+              alt="Icon presentation"
+            />
+            <h6>Forum</h6>
+            <p class="paragraph-program">
+              Liste to the models from various countries about the messages of
+              sharing and opening
+            </p>
+          </li>
+          <li class="mobile-program">
+            <img
+              class="icon-program"
+              src="./img/workshop.svg"
+              alt="Icon presentation"
+            />
+            <h6>Workshop</h6>
+            <p class="paragraph-program">
+              Liste to the models from various countries about the messages of
+              sharing and opening
+            </p>
+          </li>
+          <li class="mobile-program">
+            <img
+              class="icon-program"
+              src="./img/dharma-wheel.svg"
+              alt="Icon presentation"
+            />
+            <h6>Dharma</h6>
+            <p class="paragraph-program">
+              Liste to the models from various countries about the messages of
+              sharing and opening
+            </p>
+          </li>
+        </ul>
+        <div class="button-program">
+          <button type="button" class="btn-btn">Join Estudio Polanco</button>
+        </div>
+        <div class="link-program">
+          <a href="#">SEE THE WHOLE PROGRAM</a>
+        </div>
+      </section>
+      <section id="gallery" class="models">
+        <div class="title-models">
+          <h4>Featured Models</h4>
+        </div>
+        <hr />
+        <ul class="models-list"></ul>
+      </section>
+      <div class="button-more">
+        <button type="button">MORE &#5167;</button>
+      </div>
+      <div class="desktop-logo">
+        <div class="title-about-four">
+          <h5>Partners</h5>
+          <hr />
+        </div>
+        <div class="logos">
+          <img src="./img/Apple_logo.svg" alt="Logo" class="img-logos" />
```

Comment:
- [ ] As required, please use the correct background color and spacings regarding this section!

Reply:
- [ ] Kindly use the required colour for the footer as requested by the previous reviewers.

_See images below._

**Original** | **Your project**
  ----------- | -----------
  ![image](https://user-images.githubusercontent.com/49621279/124357571-0f85b200-dbd1-11eb-9fff-71ae9c6aff88.png) | ![image](https://user-images.githubusercontent.com/49621279/124357578-1c0a0a80-dbd1-11eb-93e0-c76e26959308.png)

Reply:
* nice work implementing some of the requested changes, but the background colour is still different from the original( color: #272A31) please use the required colour for the footer